src/main/java/kr/codesquad/model/Ladder.java

```diff
@@ -9,36 +15,67 @@ public Ladder(Random random) {
         this.random = random;
     }
 
-    public String[][] makeLadder(String joinMember, String maxLadderHeight) {
-        String[][] ladder = new String[Integer.parseInt(maxLadderHeight)][
-                Integer.parseInt(joinMember) + Integer.parseInt(joinMember) - 1];
-        makeColumn(ladder);
+    public List<List<String>> make(String joinMembers, int maxLadderHeight) {
+        List<String> joinMembersSplit = new ArrayList<>(Arrays.asList(joinMembers.split(",")));
+        List<List<String>> ladder = new ArrayList<>();
+        init(ladder, maxLadderHeight, joinMembersSplit.size());
+        Queue<String> joinMemberQueue = new LinkedList<>(joinMembersSplit);
+        makeColumn(ladder, joinMemberQueue);
```

Comment:
```suggestion
        makeColumn(ladder, joinMemberSplit);
```
만들어진 큐는 실제로는 makeColumn에서만 활용하니까 makeColumn 내부에서 만들어주면 어떨까요?